Community/alumni.md

```diff
@@ -214,15 +266,12 @@ Ore-Aruwaji Oloruntola was selected for Outreachy 2020
 Build Workflow Process for CHAOSS Diversity & Inclusion Badging
 
 
-#### Blog Posts
-
-* [Weekly Posts](https://medium.com/@darecoder/how-i-got-selected-as-an-outreachy-intern-b0c378a66bed)
-
-
-#### Social Media
-
+#### Links
 
+* [Blog Posts](https://medium.com/@darecoder/how-i-got-selected-as-an-outreachy-intern-b0c378a66bed)
 * [Twitter](https://twitter.com/thecraftman_)

```

Comment:
does it make sense to remove twitter to maintain consistency for all the projects?